src/SpectatorView.Native/SpectatorView.Compositor/Compositor/DeckLinkDevice.cpp

```diff
@@ -286,6 +288,11 @@ class OutputScheduler : public IDeckLinkVideoOutputCallback
 
     }
 
+    void SetLowLatencyMode(bool isEnabled)
+    {
+        minFramesQueued = isEnabled ? 1 : defaultMinFramesQueued;
+    }
```

Comment:
Would it be better take in a float or double here and interpolate between these two values with a slider? Or do we know that we will always want this toggle between 1 frame queued vs default frames queued?

Reply:
Personally I would probably only change between low-latency and high-safety but I can see that one might want to have finer control over this and it should be easy-enough to implement.